sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -113,9 +114,15 @@ func App() *cli.App {
 					Destination: &superpowered,
 					Value:       false,
 				},
+				&cli.BoolFlag{
+					Name:        "bootstrap",
```

Comment:
Are "superpowered" and "bootstrap" mutually-exclusive, or can you have a superpowered bootstrap container?

Reply:
Bootstrap containers can't be given the superpowered treatment (just yet), I added a validation to fail if the `superpowered` flag was passed and the container type is `bootstrap`.